src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let clone = { ...state };
+  const arrClone = [];
```

Comment:
Dont use data type names in variable names.
[NAMING] - use proper names for object copy and